code/parse/sexp.cpp

```diff
@@ -22638,6 +22657,7 @@ void multi_sexp_show_subtitle_text()
 	Current_sexp_network_packet.get_bool(center_y);
 	Current_sexp_network_packet.get_int(width_pct);
 	Current_sexp_network_packet.get_bool(post_shaded);
+	Current_sexp_network_packet.get_float(line_height_factor);
```

Comment:
I'm not super familiar with the multi_sexp code but I do believe that this does qualify as a breaking change with the current code. I think that it's handled fairly well in one direction but not the other. An older client sending the packet to a newer client will be handled okay even though the data is absent, but not the other way around since the older client will have excess data in the packet thereby triggering an assert.

I think this is fixable with a single line change to `multi_sexp_eval()`, changing `Current_sexp_network_packet.finished_callback()` to `Current_sexp_network_packet.sexp_discard_operator()`, which have the same result but with the benefit of discarding the excess data safely. Unfortunately that doesn't help current builds so this PR is still a breaking change.

There are two options that I see: 1) Don't alter the multi version of this sexp and let it work as it used to. We can then add the functionality later during the next multi bump, with an issue being opened for it as a reminder. 2) Bump the multi version for this PR but also include the change to `multi_sexp_eval()` so that such changes in the future *won't* require a version bump.

Reply:
mmf... I'd rather keep this PR simple and not bump any versions.  It's easy enough to use a default value for the sexp.  I'll open a new issue for the future multi bump.

Reply:
Created #3773

Reply:
The question of what to do if a SEXP change required new data to be sent in a packet was one I considered when writing the multi SEXP code. In the end, it seemed a little pointless to spend ages writing code to deal with it when it could never result in different versions working well with each other and standard practice amongst multi players was to make sure they all had the same build anyway.

Reply:
> and standard practice amongst multi players was to make sure they all had the same build anyway.

Which is almost certainly why it hasn't been an issue in the past. I hesitated to even suggest a multi version bump here except that we've just done so much work to stabilize multi that I want to take stuff like this a bit more seriously. And aside from versioning every single SEXP for multi use I don't really see a good way to make it safe either.

But the multi_sexp code does appear to handle things quite well for the most part. It doesn't handle incompatibility of course but does deal with it better than every other part of the multi code. It's just that `finished_callback()` is rather unpredictable on release builds and has the assert in there on debug builds (warranted obviously, but problematic here) when the packet is longer than expected.